### ISSUE TITLE
Specification: MWEs in ParlaMint

### ISSUE THREAD

**Issue by matyaskopp** [clarin-eric/ParlaMint#236]
related to: #204

I have an idea for the implementation of MWEs in ParlaMint. We use `<linkGrp>` and `<link>` for annotating the relation between pairs of words. It is not possible to use it in MWEs' situation. 

I am suggesting to use `<spanGrp>` and `<span>`. [17.3 Spans and Interpretations](https://tei-c.org/release/doc/tei-p5-doc/en/html/AI.html#AISP)

```XML
<s>
  <w XML:id="w1">Nice</w>
  <w XML:id="w2">to</w>
  <w XML:id="w3">meet</w>
  <w XML:id="w4">you</w>
  <pc>!</pc>
  <spanGrp>
    <span from="#w1" to="#w4" ana="#greeting"/>
  </spanGrp>
</s>
```

If I correctly understand TEI documentation, there is another solution to how to encode span:
```XML
  <spanGrp>
    <span target="#w1 #w2 #w3 #w4" ana="#greeting"/>
  </spanGrp>
```

I am not sure if this is the best solution and which variant `from-to`/`targer` I prefer. This is just the first draft (I am planning to use MWEs in another project, so I am raring to know the ParlaMint solution, so I will be able to reuse it)

**Comment by TomazErjavec** [comment]
My comments:
* Do we actually need MWEs in ParlaMint? For what exactly?
* We annotate MWEs in some Slovene corpora (http://hdl.handle.net/11356/1434), where we simply used linkGrp/link. The [definition](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-link.html) is "an association or hypertextual link among elements" which seems appropriate. But also note that the reason we used links is that (our) MWEs need not always involve adjecent words, i.e. they can have gaps which then forces you to list the tokens explicitly.
* But if the MWE words are always contiguous then why not follow [Parla-CLARIN](https://clarin-eric.github.io/parla-clarin/#sec-anasegment) which suggest using in-place `<seg>` elements?
